media-sound/bitwig-studio/bitwig-studio-4.3.ebuild

```diff
@@ -0,0 +1,80 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit desktop unpacker xdg
+
+DESCRIPTION="Multi-platform music-creation system for production, performance and DJing"
+HOMEPAGE="http://bitwig.com"
+SRC_URI="https://downloads.bitwig.com/stable/${PV}/${P}.deb"
+LICENSE="Bitwig"
+SLOT="0"
+KEYWORDS="~amd64"
```

Comment:
Looking at the ebuild it seems like this is a precompiled package.
In this case, please use `KEYWORDS="-* ~amd64"` to indicate that this package won't work for arches other than amd64.